css/css-text/line-breaking/line-breaking-013.xht

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+  <title>CSS Test: atomic inlines are treated as ideographic characters for line-breaking</title>
+  <meta charset="utf-8" />
+  <link rel="author" title="Mike Bremford" href="http://bfo.com/" />
+  <link rel="help" href="https://www.w3.org/TR/css-text-3/#line-break-details" />
+  <meta name="assert" content="Atomic inlines are treated as ideographic characters for line-breaking" />
+  <link rel="match" href="reference/line-breaking-013-ref.xht" />
+  <style>
+    p {
```

Comment:
Styling the `<p>` doesn't seem essential for the test. I suggest dropping it to simplify, and to avoid misleading people into thinking that it might matter somehow.